### PR TITLE
Cicd/Build docker

### PR DESCRIPTION
**Pull request by CharlieMc0**
Added CI step to build docker image

Added a script to streamline the wallet,`.env`, and faucet request for individuals following our Quickstart guide

Removed the need for Alchemy keys

Changes to docs are being made in this branch - https://github.com/zeta-chain/zeta-apps/tree/docs/update-quickstart

### CODE REVIEW

File: scripts/create-simple-env.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      `\nYour request for testnet assets has been submitted and will arrive at the ${newWallet.address} wallet soon.`
+    );
+  });
+});
+
+req.on("error", error => {
+  console.error(error);
+});
+
+req.end();
```
Comment (CharlieMc0) [inline scripts/create-simple-env.ts:49]: @mgrabina For context - this is use case I had in mind for a CLI faucet based on Denis's suggestions from his work trial to make it easier to set up a local environment for the quickstart guides. 

This works using the existing faucet.

Comment (mgrabina) [reply scripts/create-simple-env.ts:49]: @CharlieMc0 as both zetalabs and discord bot hits lambda thru apigatway endpoint so we would be able to do it here as well, we should think about rate limits and the 24h/drip validation if needed to bypass

**Comment by CharlieMc0** [comment]
@fadeev `yarn setup` feels like a required step and that it would be used to setup the yarn project which isn't accurate here. If someone wanted to use their own wallet they wouldn't use the `yarn create-simple-env` script and would instead manually edit the env file. 

If this was only ever used for tutorials I'd agree with you but since it is also used by internal teams and could also be the starting point for someone building their own app I want to make sure it's clear this script is optional and just used for a test environment, not something you'd use for real ongoing development. 

All that said, I think changing it to `yarn setup-tutorial` would work well.

**Review by fadeev — APPROVED** [review]
👍

**Pull request merged**